### PR TITLE
nbuono first commit! added learn_descriptors. just general setup

### PR DESCRIPTION
**Pull request by PizzaRoll04**
plz accept

### CODE REVIEW

File: experimental/learn_descriptors/learn_descriptors.cc [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#include <iostream>
+
+#include "learn_descriptors.hh"
+
+void robot::experimental::learn_descriptors::hello_world(const std::string &in)
```
Comment (ewfuentes) [inline experimental/learn_descriptors/learn_descriptors.cc:5]: Make the namespace separate.

File: experimental/learn_descriptors/BUILD [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+package(features=["warning_compile_flags"])
+
+cc_library(
+  name = "learn_descriptors",
+  hdrs = ["learn_descriptors.hh"],
+  visibility = ["//visibility:public"],
+  srcs = ["learn_descriptors.cc"],
+#   deps = [
+#     "@eigen",
+#   ]
```
Comment (ewfuentes) [inline experimental/learn_descriptors/BUILD:11]: delete the commented lines

File: experimental/learn_descriptors/learn_descriptors.cc [hunk 3]
```diff
@@ -0,0 +1,7 @@
+#include "learn_descriptors.hh"
+
+#include <iostream>
+
+void robot::experimental::learn_descriptors::hello_world(const std::string &in) {
```
Comment (ewfuentes) [inline experimental/learn_descriptors/learn_descriptors.cc:5]: put the namespace on a separate line, so:
```suggestion
namespace robot::experimental::learn_descriptors {
void hello_world(const std::string &in) {
```

Comment (ewfuentes) [reply experimental/learn_descriptors/learn_descriptors.cc:5]: don't forget the closing brace.

File: experimental/learn_descriptors/learn_descriptors.cc [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#include "learn_descriptors.hh"
```
Comment (ewfuentes) [inline experimental/learn_descriptors/learn_descriptors.cc:1]:
```suggestion
#include "experimental/learn_descriptors/learn_descriptors.hh"
```

Comment (ewfuentes) [reply experimental/learn_descriptors/learn_descriptors.cc:1]: I'm not sure how this worked...

**Review by ewfuentes — COMMENTED** [review]
github doesn't have the best code review interface, so repeating one of the comments. Once the changes have been made, I'm happy to merge the changes.

File: experimental/learn_descriptors/learn_descriptors.cc (hunk 4 above)
Comment (PizzaRoll04) [reply experimental/learn_descriptors/learn_descriptors.cc:1]: I believe you can use local includes if it's specified under the hdrs attribute in BUILD. I'll make it relative to root for style and consistency/clarity though

File: experimental/learn_descriptors/learn_descriptors.cc (hunk 1 above)
Comment (PizzaRoll04) [reply experimental/learn_descriptors/learn_descriptors.cc]: done in commit 6d07ea92c4fc41e38ce5a3523b2719ab84e8a9dc

File: experimental/learn_descriptors/learn_descriptors.cc (hunk 3 above)
Comment (PizzaRoll04) [reply experimental/learn_descriptors/learn_descriptors.cc]: done in commit 6d07ea92c4fc41e38ce5a3523b2719ab84e8a9dc

**Review by ewfuentes — APPROVED** [review]

**Pull request merged**